examples/example_plugin/example_plugin/ui/NonModelView.jsx

```diff
@@ -0,0 +1,10 @@
+import GenericView from "@views/generic/GenericView";
+
+
+export default function NonModelView(){
+    return (
+        <GenericView gridBackground="white">
+            <div> Im a humble 🙈 Non Model Plugin View.</div>
+        </GenericView>
+    )
```

Comment:
"Non-Model view" is not super self-explanatory. Perhaps something like "client-side-only view" might be clearer?